packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -102,18 +112,35 @@ mkdir -p %{buildroot}/%{_prefix}/lib/gofer/plugins
 cp etc/gofer/plugins/katelloplugin.conf %{buildroot}/%{_sysconfdir}/gofer/plugins
 cp src/katello/agent/katelloplugin.py %{buildroot}/%{_prefix}/lib/gofer/plugins
 
-mkdir -p %{buildroot}/%{_prefix}/lib/yum-plugins
-cp src/yum-plugins/package_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
-cp src/yum-plugins/enabled_repos_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
+%if %{dnf_install}
+%define katello_libdir %{python3_sitelib}/katello
+%define plugins_dir %{python3_sitelib}/dnf-plugins
+%define plugins_confdir %{_sysconfdir}/dnf/plugins
+%else
+%define katello_libdir /%{python_sitelib}/katello
```

Comment:
Is that `/` before `%{python_sitelib}` intentional?

Reply:
interestingly, el5 build breaks without it

Reply:
Something is fishy about it. Taking a look